src/mscorlib/shared/System/Collections/Concurrent/ConcurrentQueue.cs

```diff
@@ -1078,14 +1111,15 @@ internal struct Slot
                 public int SequenceNumber;
             }
         }
-    }
 
-    /// <summary>Padded head and tail indices, to avoid false sharing between producers and consumers.</summary>
-    [DebuggerDisplay("Head = {Head}, Tail = {Tail}")]
-    [StructLayout(LayoutKind.Explicit, Size = 3*Internal.PaddingHelpers.CACHE_LINE_SIZE)] // padding before/between/after fields
```

Comment:
@jkotas how about for StructLayout here? I assume the old size of `3*Internal.PaddingHelpers.CACHE_LINE_SIZE` should not change to hardcoded 384 either.

Therefore, I'll just keep the existing diff in coreclr for `PaddedHeadAndTail`